wiki/Modding/Nomination/en.md

```diff
@@ -0,0 +1,30 @@
+---
+tags:
+  - nominations
+  - nom
+  - noms
+  - heart
```

Comment:
```suggestion
```
Hearts are typically associated with [qualification](https://osu.ppy.sh/wiki/en/Beatmap_ranking_procedure#qualification), not so much nominations.